workspaces/orchestrator/plugins/orchestrator-backend/src/helpers/filterBuilder.ts

```diff
@@ -221,9 +251,13 @@ function convertToBoolean(value: any): boolean {
 function formatValue(
   fieldName: string,
   fieldValue: any,
-  fieldDef: IntrospectionField,
+  fieldDef: IntrospectionField | undefined,
   type: ProcessType,
 ): string {
+  if (!fieldDef) {
+    return `"${fieldValue}"`;
+  }
```

Comment:
if `fieldDef` is undefined, it means that there is no such field in the introspection field list and so we don't know its format (i.e. string, date, ...).
Converting to string can be wrong.